api/python/quilt3/main.py

```diff
@@ -312,18 +335,39 @@ def create_parser():
     disable_telemetry_p.set_defaults(func=cmd_disable_telemetry)
 
     # install
-    shorthelp = "Install a package"
-    install_p = subparsers.add_parser("install", description=shorthelp, help=shorthelp, allow_abbrev=False)
+    shorthelp = "Install a package or a collection of packages"
+    install_p = subparsers.add_parser(
+        "install",
+        description=shorthelp,
+        help=shorthelp,
+        allow_abbrev=False,
+        formatter_class=argparse.RawTextHelpFormatter
+    )
     install_p.add_argument(
         "name",
-        help=(
-            "Name of package, in the USER/PKG[/PATH] format ([/PATH] is deprecated, use --path parameter instead)"
-        ),
-        type=str,
+        help="Name can be of two representations:\n"
+             "- Name of package, in the USER/PKG[/PATH] format ([/PATH] is deprecated, use --path parameter instead)\n"
+             "- Path to quilt.yaml or quilt.yml package dependency file.\n"
+             "  Syntax:\n"
```

Comment:
This is a fine start to documenting the file format... but it doesn't belong in CLI help :) It belongs in actual docs in markdown.

Reply:
I see